models/yang/annotations/openapi_annotations/acme-system-annot.yang

```diff
@@ -0,0 +1,17 @@
+module acme-system-annot { 
```

Comment:
Why is this file being added?

Reply:
I have deleted the file from the branch as it's not needed.